roles/neutron-common/templates/etc/neutron/policy.json

```diff
@@ -1,14 +1,13 @@
 {
-    "context_is_admin": "role:admin or role:cloud_admin",
-    "cloud_admin_privilege": "role:admin or role:cloud_admin",
-    "is_owner": "project_id:%(tenant_id)s or project_id:%(project_id)s",
-    "admin_or_owner": "rule:cloud_admin_privilege or rule:is_owner or rule:project_admin_privilege",
+    "context_is_admin": "role:admin",
+    "owner": "tenant_id:%(tenant_id)s or project_id:%(tenant_id)s or project_id:%(project_id)s",
+    "projectadmin": "role:project_admin and rule:owner",
+    "admin_or_cloudadmin_or_projectadmin": "rule:context_is_admin or role:cloud_admin or rule:projectadmin",
+    "admin_or_owner": "rule:admin_or_cloudadmin_or_projectadmin or rule:owner",
     "context_is_advsvc":  "role:advsvc",
-    "admin_or_network_owner": "rule:admin_or_owner or tenant_id:%(network:tenant_id)s",
-    "admin_only": "role:admin",
-    "project_admin_privilege": "(project_id:%(tenant_id)s and role:project_admin) or rule:horizon_project_admin_privilege",
-    "horizon_project_admin_privilege": "(project_id:%(project_id)s and role:project_admin)",
-    "admin_or_admin_owner": "rule:cloud_admin_privilege or rule:project_admin_privilege",
+    "admin_or_network_owner": "rule:admin_or_cloudadmin_or_projectadmin or tenant_id:%(network:tenant_id)s",
```

Comment:
Network owner check may need `project_id` checks similar to that of `rule:owner`. This will also need to be tested before final PR.